tests/integration/golang/aim/dashboard/create_dashboard_test.go

```diff
@@ -69,9 +69,13 @@ func (s *CreateDashboardTestSuite) Test_Ok() {
 				&resp,
 			)
 			assert.Nil(s.T(), err)
-			assert.Equal(s.T(), tt.requestBody.Name, resp.Name)
-			assert.Equal(s.T(), s.app.Type, resp.Type)
-			assert.Equal(s.T(), tt.requestBody.AppID, resp.AppID)
+
+			dashboards, err := s.dashboardFixtures.GetDashboards(context.Background())
+			assert.Nil(s.T(), err)
+			assert.Equal(s.T(), dashboards[0].Name, resp.Name)
```

Comment:
I'd say we need to change the assertion params for these -- the first param is the expectation, which is `tt.requestBody.Name` in this case, and the second would be the thing under test `dashboards[0].Name`